griptape/tasks/image_generation_task.py

```diff
@@ -0,0 +1,36 @@
+from attr import define, field, Factory
+from griptape.artifacts import ImageArtifact
+from griptape.drivers import (
+    BaseImageGenerationDriver,
+    Dalle2ImageGenerationDriver,
+)
+from griptape.engines.image_generation.base_image_generation_engine import (
+    BaseImageGenerationEngine,
+)
+from griptape.engines.image_generation.prompt_image_generation_engine import (
+    PromptImageGenerationEngine,
+)
+from griptape.tasks import BaseTextInputTask
+
+
+@define
+class ImageGenerationTask(BaseTextInputTask):
+    image_generation_driver: BaseImageGenerationDriver = field(
+        default=Factory(lambda: Dalle2ImageGenerationDriver()), kw_only=True
+    )
```

Comment:
Should we remove this to simplify the interface? The user can always pass a custom driver into `image_generation_engine`.